src/components/AssetsFilters/index.jsx

```diff
@@ -41,7 +19,7 @@ export const AssetsFilters = ({ assetsFilters, updateFilter, updatePage }) => (
             id={type.key}
             name={type.key}
             label={type.displayName}
-            checked={assetsFilters[type.key]}
+            checked={assetsFilters.assetTypes[type.key]}
```

Comment:
assetsFilters is an object with a key assetTypes, which is a dictionary of key-value pairs "filter": "is-filter-enabled". This is so that we can leave room for other kinds of filters in the future.